_posts/2021-02-02-SUGCH-2021-1.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: SUGCH 2021/1 - Marketing Automation Building Blocks
+---
+
+Our first online meetup for 2021 took place on 02. February 2021. We had 20 attendees on this event and even some international guests which is one of the benefits of online meetups.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/sugch?src=hash&amp;ref_src=twsrc%5Etfw">#sugch</a> 1/2021 ready to kick off <a href="https://t.co/rO1f02cLuU">pic.twitter.com/rO1f02cLuU</a></p>&mdash; Mark Lowe (@misterlow) <a href="https://twitter.com/misterlow/status/1356618494687793153?ref_src=twsrc%5Etfw">February 2, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Presentations
+
+### Marketing Automation Building Blocks
+
+Our guests [Rashmy Patwari](https://twitter.com/nrashmy) and [Ozell McBride](https://twitter.com/omcbri1), both from the United States, showed us the building blocks involved in creating a Marketing Automation campaign.  
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Let&#39;s learn all about Marketing Automation Building Blocks by <a href="https://twitter.com/nrashmy?ref_src=twsrc%5Etfw">@nrashmy</a> and <a href="https://twitter.com/omcbri1?ref_src=twsrc%5Etfw">@omcbri1</a> <a href="https://t.co/CReYuDwW1c">pic.twitter.com/CReYuDwW1c</a></p>&mdash; SUG CH (@sugch) <a href="https://twitter.com/sugch/status/1356620556217552896?ref_src=twsrc%5Etfw">February 2, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+In the first part of the presentation, Ozell covered importing XDB Contacts from CSV into Contact Lists and gave a quick introduction to Sitecore Forms and its Submit Actions. Rashmi then took over and showed us how to create an Engagement Plan that send e-mails through EXM and created lists based on goals all in all covering several building blocks of the Sitecore Experience Platform.
```

Comment:
```suggestion
In the first part of the presentation, Ozell covered importing xDB contacts from CSV into a contact list in the List Manager and gave a quick introduction to Sitecore Forms and its Submit Actions. Rashmi then took over and showed us how to create a Marketing Automation plan that sends e-mails through EXM and creates a list based on a triggered goal. All in all covering several building blocks of the Sitecore Experience Platform.
```

Reply:
Thanks! Super fast and to the point. Just minor suggestions regarding the product names. :)